faq/prebid-server-faq.md

```diff
@@ -77,18 +77,94 @@ creates or updates the `uids` cookie.
 the hosting company in some scenarios, but it's technically unavoidable and better for both buyers and sellers to have one ID available rather than zero.
+
+## How does Prebid Server support privay signals?
+
+### Mobile 'Limit Ad Tracking' flag
+
+If PBS receives 'device.lmt' flag in the OpenRTB request, it does the following anonymization:
+
+- Mask take off the last byte of the IPv4 address and the last 2 bytes of IPv6 addresses
+- Removes user.id and user.buyeruid
+- Removes the request.device.ifa attribute
+- Rounds the request.device.geo. {lat,lon} to two decimal places
+
+### GDPR
+
+Prebid Server host companies and publishers have the ability to control the enforcement
+activities that take place.
+
+The enforcement strategy changed significantly between TCF 1.1 and TCF 2.0. [TCF2](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md) is a 
+more nuanced and stricter policy.
+
+#### TCF 1.1
+
+If Prebid Server determines that the user is in GDPR scope and doesn't consent
+to *all* of the vendor's 'purposes' as declared in the Global Vendor List, it 'anonymizes'
+the request to the adapters:
+
+- Mask take off the last byte of the IPv4 address and the last 2 bytes of IPv6 addresses
+- Removes user.id and user.buyeruid
+- Removes the request.device.ifa attribute
+- Rounds the request.device.geo. {lat,lon} to two decimal places
+
+Full details are available [here](https://github.com/rubicon-project/prebid-server-java/blob/master/docs/developers/PrebidServerJava_GDPR_Requirements.pdf).
+
+#### TCF 2.0
+
+If Prebid server determines the user is in GDPR scope, then consent is independently tested
+for each 'Purpose' with different consequences for each:
+
+{: .table .table-bordered .table-striped }
+| TCF Purpose | Consequence of Not Obtaining Consent |
+| ----------- | ------------------------------------ |
+| 1 - Device Access | Prevents one or more usersync activities for one or more vendors. |
+| 2 - Basic Ads | May result in skipping one or more bid adapters in the auction. |
+| 4 - Personalized Ads | May result in removing the userIds before calling one or more bid adapters. |
+| 7 - Measure Ad Performance | May result in skipping one or more analytics adapters. |
+| Special Feature 1 - Use precise geolocation data | May result in rounding lat/long values and IP address before sending to server-side adapters. |
+
+{: .alert.alert-danger :}
+Note: Support for TCF purposes other than Device Access is still under development and is
+expected to be released in May 2020.
+
+More details are available [here](https://docs.google.com/document/d/1fBRaodKifv1pYsWY3ia-9K96VHUjd8kKvxZlOsozm8E/edit#).
+
+### COPPA
+
+The [Children's Online Privacy Protection Act (COPPA)](https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule) is a rule in the US.
```

Comment:
We should provide minimal detail on what this law covers: 

The [Children's Online Privacy Protection Act (COPPA)](https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule) is a law in the US which imposes certain requirements on operators of websites or online services directed to children under 13 years of age, and on operators of other websites or online services that have actual knowledge that they are collecting personal information online from a child under 13 years of age.